{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/pytest-dev/pytest-faulthandler",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-faulthandler/pytest-faulthandler-1.5.0.tar.gz",
    "subdirs": [
      "linux-64"
    ],
    "summary": "py.test plugin that activates the fault handler module for tests",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "1.5.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003",
    "cf201901"
  ],
  "package": "pytest-faulthandler-1.5.0-py27_0.tar.bz2",
  "repodata": {
    "build": "py27_0",
    "build_number": 0,
    "depends": [
      "faulthandler",
      "pytest",
      "python 2.7*"
    ],
    "license": "MIT",
    "md5": "fec9fe7647cd77c6e4b09b182d09a496",
    "name": "pytest-faulthandler",
    "sha256": "9b4471c2e77eba53033c3b6d36c8ed399d6fd509d51df19c42d3b906eb068ab7",
    "size": 8203,
    "subdir": "linux-64",
    "version": "1.5.0"
  },
  "repodata_version": 1,
  "subdir": "linux-64",
  "url": "https://github.com/regro/releases/releases/download/linux-64/pytest-faulthandler-1.5.0-py27_0.tar.bz2/pytest-faulthandler-1.5.0-py27_0.tar.bz2"
}